src/diffusers/pipelines/audioldm2/pipeline_audioldm2.py

```diff
@@ -503,6 +562,15 @@ def encode_prompt(
                     negative_prompt_embeds = negative_prompt_embeds[:, None, :]
                     # make sure that we attend to this single hidden-state
                     negative_attention_mask = negative_attention_mask.new_ones((batch_size, 1))
+                elif text_encoder.config.model_type == "vits":
```

Comment:
[Use a single variable to check if it's a ViTS model](https://github.com/huggingface/diffusers/pull/5381/files#r1542416708)